support/sys-stat.c

```diff
@@ -76,8 +80,10 @@ Mono_Posix_ToStat (void *_from, struct Mono_Posix_Stat *to)
 	to->st_gid        = from->st_gid;
 	to->st_rdev       = from->st_rdev;
 	to->st_size       = from->st_size;
-	to->st_blksize    = from->st_blksize;
-	to->st_blocks     = from->st_blocks;
+#ifndef HOST_WIN32
+	to->st_blksize = from->st_blksize;
```

Comment:
...though it would be nice if you didn't change the vertical alignment of the `=`s here. ;-)